cpp/test/Ice/servantLocator/TestAMD.ice

```diff
@@ -28,8 +28,8 @@ exception TestImpossibleException
 
     void unknownExceptionWithServantException();
 
-    string impossibleException(bool throw) throws TestImpossibleException;
-    string intfUserException(bool throw) throws TestIntfUserException, TestImpossibleException;
+    string impossibleException(["cpp:identifier:_cpp_throw"] bool throw) throws TestImpossibleException;
```

Comment:
I would not use leading underscores for the new C++ identifiers.

Reply:
I renamed all the identifiers outside of the `escape` test.

If you really want, I can also do the `escape` test, but this will render the file utterly unreviewable.
And I don't think the names for that test matters, since that test is already using insane names like `break` and `return`.